accounts-db/src/append_vec.rs

```diff
@@ -1130,7 +1141,10 @@ pub mod tests {
         let now = Instant::now();
         let mut sizes = vec![];
         for sample in 0..size {
-            let account = create_test_account(sample);
+            let mut account = create_test_account(sample);
+            if account.1.lamports() == 0 {
+                account.1.set_lamports(1); // non-zero, so this is not a default default account
+            }
```

Comment:
because scan accounts skips accounts which are 'default'...

and this test loops through test accounts with `sample` sometimes = 0 which means 0 lamport accounts. Yuck. I'm not sure of the best way to fix this. maybe add +1 to `sample` in the test account creation method so 0 doesn't create default default accounts (with default pubkey, too).